README.md

```diff
@@ -52,16 +52,16 @@ If you wish to contribute to this template, the following information may be hel
 In ```PushStarter/AppDelegate.swift``` you register for notification as below:
 
 ```Swift
-func application(application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: NSData) {
-  FH.pushRegister(deviceToken, success: { res in // [1]
-    let notification = NSNotification(name: "sucess_registered", object: nil)
-    NSNotificationCenter.defaultCenter().postNotification(notification)
-    print("Unified Push registration successful")
-  }, failure: {failed in                         // [2]
-    let notification = NSNotification(name: "error_register", object: nil)
-    NSNotificationCenter.defaultCenter().postNotification(notification)
-    print("Unified Push registration Error \(failed.error)")
-  })
+func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
```

Comment:
I think we could simplify the README and only put:
```swift
    func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
        FH.pushRegister(deviceToken: deviceToken, success: { res in
            print("Unified Push registration successful")
        }, error: {failed in
            print("Unified Push registration Error \(failed.error)")
        })
    }
```
No need to clutter it with detailed information on how we displayed  the error/success message on the UI (through Notification centre redirection)